src/modules/mainFunction.js

```diff
@@ -64,26 +66,58 @@ class TaskList {
     this.tasksContainer.addEventListener('click', (e) => {
       const taskItem = e.target.closest('.taskItems');
       if (taskItem) {
-        const infoTask = taskItem.querySelector('.infoTask');
-        infoTask.addEventListener('blur', () => {
-          const newDescription = infoTask.value.trim();
+        const descriptionElement = taskItem.querySelector('.infoTask');
+        descriptionElement.addEventListener('blur', () => {
+          const newDescription = descriptionElement.value.trim();
           if (newDescription) {
-            const id = parseInt(infoTask.dataset.index, 10);
+            const id = parseInt(descriptionElement.dataset.index, 10);
             const taskIndex = this.tasksInfo.findIndex((task) => task.index === id);
             this.tasksInfo[taskIndex].description = newDescription;
           }
           this.display();
+          this.checkStatus();
           this.saveTasks();
         });
       }
     });
   }
 
+  checkStatus = () => {
```

Comment:
- [ ] It's one of the project requirements to create a new JavaScript file that will contain methods related to the status updates (completed: true/false). Kindly create a new file for the checkStatus method and imported it as a module. ✍🏻 
- [ ] Make sure to add an  (change) event listener to the checkbox.